src/site/content/en/blog/mouse-acceleration/index.md

```diff
@@ -0,0 +1,175 @@
+  } else {
+    console.log("pointer is unlocked");
+  }
+});
+
+document.addEventListener("pointerlockerror", () => {
+  console.log("pointer lock error");
+});
+```
+
+### Disable mouse acceleration {: #disable-mouse-acceleration }
+
+Call `requestPointerLock()` with  `{ unadjustedMovement: true }` to disable
+OS-level adjustment for mouse acceleration, and access raw mouse input.
+This way, mouse movement data from `mousemove` events won't include mouse
+acceleration when the pointer is locked.
+
+Use the new returned promise from `requestPointerLock()` to know if the request
+was successful.
+
+```js
+function requestPointerLockWithUnadjustedMovement() {
+  const promise = myTargetElement.requestPointerLock({
+    unadjustedMovement: true,
+  });
+
+  if (!promise) {
+    console.log("disabling mouse acceleration is not supported");
+    return;
+  }
+
+  return promise
+    .then(() => console.log("pointer is locked"))
+    .catch((error) => {
+      if (error.name === "NotSupportedError") {
+        // Some platforms may not support unadjusted movement.
+        // You can request again a regular pointer lock.
+        return myTargetElement.requestPointerLock();
+      }
+    });
+}
+```
+
+It is possible to toggle between accelerated and non-accelerated mouse movement
+data without releasing the pointer lock. Simply request the pointer lock again
+with the desired option. If that request fails, the original lock will remain
+intact and the returned promise will reject. No pointer lock events will fire
+for a failed change request.
+
+### Browser support {: #browser-support }
+
+The Pointer Lock API is [well supported across browsers]. However Chromium-based
+browsers (e.g. Chrome, Edge, etc.) are the only ones to support disabling
+OS-level adjustment for mouse acceleration at the time of writing.
+
+### Platform support {: #platform-support }
+
+Disabling OS-level adjustment for mouse acceleration is supported on Chrome OS,
+macOS Catalina 10.15.1, and Windows. Linux will follow.
+
+## Sample  {: #sample }
+
+You can play with the Pointer Lock API by running the [sample] on Glitch. Be
+sure to [check out the source code].
+
+{% Glitch { id: 'unadjusted-movement', path: 'script.js' } %}
+
+## Helpful links {: #helpful }
+
+- [Specification PR](https://github.com/w3c/pointerlock/pull/49)
+- [GitHub repository](https://github.com/w3c/pointerlock)
+- [ChromeStatus entry](https://www.chromestatus.com/feature/5723553087356928)
+- [Chrome tracking bug](https://bugs.chromium.org/p/chromium/issues/detail?id=982379)
+- [Intent to ship](TODO)
+- [Mozilla's position](https://github.com/mozilla/standards-positions/issues/448)
+- [WebKit's position](https://lists.webkit.org/pipermail/webkit-dev/2020-October/031473.html)
+
+## Acknowledgements
+
+Thanks to [James Hollyer], [Thomas Steiner], and [Joe Medley] for their reviews
+of this article.
+
+<!-- lint disable definition-case -->
```

Comment:
I'm fine with disabling this lint check if you want to go into the lint config file

Reply:
Yep!
https://github.com/GoogleChrome/web.dev/pull/4115